calcom/cal.com
# fix: Cleanup after removing connected calendar

## What does this PR do?


Fixes the database inconsistency caused when a calendar is being deleted, as even after being deleted it was still present in the SelectedCalendars table.

Fixes #9988 



https://github.com/calcom/cal.com/assets/47187878/61f9f865-41ef-4753-82de-7a89b94efde1



## Type of change



- Bug fix (non-breaking change which fixes an issue)

## How should this be tested?



- Connect a calendar
- Check that it is present in the database's SelectedCalendar table
- delete it
- See that it is also removed from the SelectedCalendar table

## Mandatory Tasks
